src/components/empty_prompt/empty_prompt.stories.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { EuiEmptyPrompt, EuiEmptyPromptProps } from './empty_prompt';
+
+import { EuiButton, EuiButtonEmpty } from '../button';
+import { EuiTitle } from '../title';
+import { EuiLink } from '../link';
+import { EuiImage } from '../image';
+import { EuiPageTemplate } from '../page_template';
+import illustration from '../../../src-docs/src/images/empty-prompt/illustration.svg';
+
+const meta: Meta<EuiEmptyPromptProps> = {
+  title: 'EuiEmptyPrompt',
+  component: EuiEmptyPrompt,
+  parameters: {
+    controls: {
+      exclude: ['data-test-subj'],
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<EuiEmptyPromptProps>;
+
+export const Playground: Story = {
+  args: {
+    title: <h2>Start adding cases</h2>,
+    layout: 'horizontal',
+    iconType: 'logoSecurity',
+    hasBorder: false,
+    hasShadow: false,
+    body: <p>Add a new case or change your filter settings.</p>,
```

Comment:
[very optional] Removing the `<p>` and changing this to a string allows devs to modify the body text dynamically. Ideally, storybook would have a JSX editor (https://github.com/storybookjs/storybook/issues/11428) that would be as easy to edit as a string editor, but alas 🥲 

```suggestion
    body: 'Add a new case or change your filter settings.', // Should be wrapped in a `<p>` in production usage, but using a string makes this easier to edit in Storybok controls
```

Reply:
> Ideally, storybook would have a JSX editor
This is something I searched far and wide for! It's very popular request and I'm surprised it hasn't come yet.